bessctl/cli.py

```diff
@@ -345,6 +345,10 @@ def find_cmd(self, line):
             return matched[0]
 
         elif len(matched) >= 2:
+            for m in  matched: # return if exact match exists
```

Comment:
Trivial, nitty: there's a double space here (`for m in<space><space>matched`).

Q (also pretty trivial): should we capture `line.strip()` once instead of calling it once per match?

More seriously: it seems like a bad design in general to have complete commands that are prefixes of longer commands. But maybe we have to allow it in general anyway.

Reply:
@chris3torek It was tab 😈...

About the line.strip(), the space at the end here plays a important role, deciding whether it is the end of a token or partial of a token. The design-wise not super clean. ;ㅁ;